### DISCUSSION TITLE
Bootstrap Additional Clusters - Azure Repos

### DISCUSSION THREAD

**Discussion by thepaulmacca** [fluxcd/flux2#2143] [Q&A]
Although bootstrapping is *much* easier with a GitHub repo, unfortunately we're still having to use Azure Repos for the moment 😐

Can anyone show me an example of how I would bootstrap additional clusters if using Azure Repos please? I have one prod cluster all set up, so would just like an example on how to now bootstrap an additional cluster down the line if needed (possibly multi-region in future, but still in a POC phase with this)

Thanks

**Comment by stefanprodan** [comment]
Please see the docs on [Using Flux on Azure](https://fluxcd.io/docs/use-cases/azure/).

**Reply by thepaulmacca** [reply]
Yep, I've been reading that but what if existing `GitRepository` and `Kustomization` objects are already in place?

I'm guessing it's some kind of limitation when using Azure Repos, as it seems a lot more straightforward with a GitHub repo

**Reply by stefanprodan** [reply]
> Yep, I've been reading that but what if existing GitRepository and Kustomization objects are already in place?

Bootstrap doesn't copy things from clusters to your repo. No matter how you bootstrap Flux (manually or with the CLI) you still need to  export your resources and place them in Git. Check out `flux export` commands, they can give you the YAMLs.

**Reply by thepaulmacca** [reply]
I'll take a look, thanks
